zebrad/tests/acceptance.rs

```diff
@@ -1835,7 +1835,7 @@ fn lightwalletd_integration_test(test_type: TestType) -> Result<()> {
 
         if test_type.needs_lightwalletd_cached_state() {
             lightwalletd
-                .expect_stdout_line_matches("Done reading [0-9]{1,7} blocks from disk cache")?;
+                .expect_stdout_line_matches("Done reading [0-9]{7} blocks from disk cache")?;
```

Comment:
This change will make some tests fail until all the `lightwalletd` cached states are synced properly again. Can you split it into a separate PR, and we'll merge it a week or so after this PR merges?

Until the fix merges, and all the PRs are updated, the fixed code will be fighting with the buggy code to make cached states. If the buggy code wins, this check will fail, and every PR that has the fix will fail.

(We used to do a full sync before every test, so doing it in the same PR wouldn't have been a problem. But that got too slow, so now we only do the full sync every week. So we have to do some fixes in stages. Arya and I both missed this when we wrote the ticket.)

Reply:
Hmm, actually, I think I have a better solution, see my comment below.

So there's no action needed here.